render-wasm/src/render/surfaces.rs

```diff
@@ -137,11 +140,44 @@ impl Surfaces {
             .draw(self.canvas(to), (0.0, 0.0), sampling_options, paint);
     }
 
+    /*
     pub fn apply_mut(&mut self, ids: &[SurfaceId], mut f: impl FnMut(&mut skia::Surface)) {
+        performance::begin_measure!("apply_mut::slice");
         for id in ids {
             let surface = self.get_mut(*id);
             f(surface);
         }
+        performance::begin_measure!("apply_mut::slice");
+    }
+    */
+
+    pub fn apply_mut(&mut self, ids: u32, mut f: impl FnMut(&mut skia::Surface)) {
```

Comment:
I love this but for me it was a bit difficult to understand

I think it would be interesting explaining that now surfaceId is working like a bit mask
```
enum SurfaceId {
    Target = 1 << 0,         // 0000_0001
    Current = 1 << 1,        // 0000_0010
    Cache = 1 << 2,          // 0000_0100
    Fills = 1 << 3,          // 0000_1000
    Strokes = 1 << 4,        // 0001_0000
    InnerShadows = 1 << 5,   // 0010_0000
    DropShadows = 1 << 6,    // 0100_0000
    Debug = 1 << 7,          // 1000_0000
}
```
And that this kind of operations `ids & SurfaceId::Target as u32` just check if that bit is active